google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/sdk/CloudSdkPanel.java

```diff
@@ -218,7 +218,12 @@ public void apply() throws ConfigurationException {
     }
     sdkServiceUserSettings.setUserSelectedSdkServiceType(selectedCloudSdkServiceType);
 
+    boolean previousAutomaticUpdateEnabled = sdkServiceUserSettings.getEnableAutomaticUpdates();
```

Comment:
I know it wasn't named in this pr, but getEnableAutomaticUpdates -> isAutomaticUpdateEnabled ?

Reply:
Agree, that's easier to read.